package.json

```diff
@@ -23,15 +24,21 @@
   },
   "homepage": "https://github.com/Kinto/kinto-discovery-lib/",
   "devDependencies": {
+    "babel-cli": "^6.5.1",
+    "babel-core": "^6.5.1",
+    "babel-plugin-transform-es2015-modules-commonjs": "^6.5.2",
+    "babel-preset-es2015": "^6.3.13",
+    "babelify": "^7.2.0",
+    "browserify": "^13.0.0",
     "chai": "^3.2.0",
     "chai-as-promised": "^5.0.0",
     "eslint": "^1.2.0",
     "mocha": "^2.4.5",
     "node-localstorage": "^1.1.2",
-    "sinon": "^1.14.1"
+    "sinon": "^1.14.1",
+    "uglifyify": "^3.0.1"
   },
   "dependencies": {
-    "babel": "^5.8.19",
     "isomorphic-fetch": "^2.2.1",
     "kinto": "^1.2.0",
```

Comment:
I don't think we are using kinto anymore.
However it is not related to this PR but I just noticed it.